src/main/java/frc/robot/MaxCalc.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot;
+
+//Recycled Class from 2019 to prevent false values being inputed into the motor controller.
+public class MaxCalc {
+    public static double limit(double value, double min, double max)
+    {
+        if(value >= max)
+        {
+            return max;
+        }
+        if(value <= min)
+        {
+            return min;
+        }
+        return value;
+    }
+}
```

Comment:
This name is too specific. No need to make a whole class just for one utility function. Maybe name this file `Utility.java` and it can be a place for us to stash other helpful functions in the future